src/points.jl

```diff
@@ -17,28 +17,35 @@ O = Point(0.0, 0.0) # origin of 2D Euclidean space
 
 ### Notes
 
-- Type aliases are `Point2`, `Point3`, `Point2f`, `Point3f`
+- Type aliases are `Point1`, `Point2`, `Point3`, `Point1f`, `Point2f`, `Point3f`
 """
 struct Point{Dim,T}
   coords::SVector{Dim,T}
+  Point{Dim,T}(coords::AbstractVector) where {Dim,T} = new{Dim,T}(coords)
 end
 
 # convenience constructors
 Point{Dim,T}(coords::NTuple{Dim,V}) where {Dim,T,V} = Point{Dim,T}(SVector(coords))
-Point{Dim,T}(coords::Vararg{V,Dim}) where {Dim,T,V} = Point{Dim,T}(SVector(coords))
+Point{Dim,T}(coords...) where {Dim,T} = Point{Dim,T}(coords)
```

Comment:
So we need to remove the explicit `Vararg{V,Dim}` specification here? Could we preserve it?